platforms/android/library-compose/src/main/java/io/element/android/wysiwyg/compose/RichTextEditor.kt

```diff
@@ -14,25 +14,31 @@ import androidx.compose.ui.viewinterop.AndroidView
 import androidx.core.content.ContextCompat
 import androidx.core.widget.addTextChangedListener
 import io.element.android.wysiwyg.EditorEditText
-import io.element.android.wysiwyg.compose.internal.ViewConnection
+import io.element.android.wysiwyg.compose.internal.ViewAction
 import io.element.android.wysiwyg.compose.internal.toStyleConfig
 import io.element.android.wysiwyg.utils.RustErrorCollector
-import io.element.android.wysiwyg.view.models.InlineFormat
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
 
 /**
  * A composable rich text editor.
  *
  * This composable is a wrapper around the [EditorEditText] view.
  *
+ * To use within a subcomposition, set the [registerStateUpdates] parameter to false.
+ *
  * @param state The state holder for this composable. See [rememberRichTextEditorState].
  * @param modifier The modifier for the layout
+ * @param registerStateUpdates If true, register the state for updates.
  * @param style The styles to use for any customisable elements
  * @param onError Called when an internal error occurs
  */
 @Composable
 fun RichTextEditor(
     state: RichTextEditorState,
```

Comment:
would it make sense to have a default value here?
```suggestion
    state: RichTextEditorState = rememberRichTextEditorState(),
```

Reply:
Actually I see that some default params are in the `private fun RealEditor` composable. They should probably be hoisted to the public one.